packages/sampler/package.json

```diff
@@ -8,27 +8,28 @@
     "interlink": "npm link ../core ../i18n ../spotlight ../ui ../webos",
     "lint": "enact lint --strict",
     "test": "echo \"No test suite\" && exit 0",
-    "pack": "build-storybook -c .storybook -o dist",
-    "serve": "start-storybook -p 8080 -c .storybook --ci"
+    "pack": "storybook build -c .storybook -o dist",
+    "serve": "storybook dev -p 8080 -c .storybook --ci"
   },
   "license": "Apache-2.0",
   "dependencies": {
     "@enact/core": "^4.9.0-alpha.3",
     "@enact/i18n": "^4.9.0-alpha.3",
     "@enact/spotlight": "^4.9.0-alpha.3",
-    "@enact/storybook-utils": "^5.1.4",
+    "@enact/storybook-utils": "enactjs/storybook-utils#feature/storybook8",
```

Comment:
After Merging https://github.com/enactjs/storybook-utils/pull/71, would you please update the branch to `feature/enact_v5`?
Also, please update the version of storybook modules to 8.1.6.